config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :flights, only: [:index, :show] do
+    member do
+      delete '/destroy/:passenger_id', action: :destroy, as: 'destroy'
+    end
```

Comment:
When you use this `member` route and run a `rails routes`, this is what gets produced: `delete "/flights/:id/destroy/:passenger_id", to: "flights#destroy"` which is not a RESTful route and therefore incorrect.

You should be using nested resource routing.